server/lib/services/noteActionHandler.js

```diff
@@ -0,0 +1,31 @@
+// const ServiceError = require('../ServiceError')
```

Comment:
Try to not commit commented code. If you are not yet ready to have this code in`master` - commit it but to another branch. You can even ask to review this uncomplete branch, but always make it explicit.